src/selectors/entities/posts.js

```diff
@@ -190,10 +207,12 @@ function formatPostInChannel(post, previousPost, index, allPosts, postIds, curre
 export function makeGetPostsInChannel() {
     return createSelector(
         getAllPosts,
+        getPostsInThread,
         (state, channelId) => state.entities.posts.postsInChannel[channelId],
         getCurrentUser,
         getMyPreferences,
-        (allPosts, postIds, currentUser, myPreferences) => {
+        (state, channelId, numPosts) => numPosts || Posts.POST_CHUNK_SIZE,
```

Comment:
This is essentially the `postVisibility` value from the web app. It will change when the post visibility changes (some clicks load more messages, or a post is added to the channel while a user is in that channel) but it won't change very often and will change at the same time as posts changing so the effect is negligible